packages/dds/merge-tree/package.json

```diff
@@ -104,10 +86,12 @@
 		"test": "npm run test:mocha",
 		"test:benchmark:report": "mocha \"dist/test/*.perf.spec.*js\" --node-option unhandled-rejections=strict,expose-gc --exit --perfMode --fgrep @Benchmark -r @fluid-internal/mocha-test-setup --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha --ignore \"dist/test/types/*\" --recursive dist/test",
+		"test:mocha": "npm run test:mocha:cjs && npm run test:mocha:esm",
+		"test:mocha:cjs": "mocha  --recursive \"dist/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:esm": "mocha  --recursive \"lib/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
```

Comment:
@anthony-murphy, @Abe27342 - I found several file containing mocha tests that did not use the *.spec* convention.  I've renamed these.  (Might be worth double checking that I found them all.)